syft/frameworks/crypten/model.py

```diff
@@ -0,0 +1,179 @@
+from typing import List
+
+import torch
+
+import syft
+
+from syft.workers.abstract import AbstractWorker
+from syft.workers.base import BaseWorker
+from syft.frameworks.crypten import utils
+
+from syft.generic.pointers.object_pointer import ObjectPointer
+from syft.generic.abstract.sendable import AbstractSendable
+
+from syft_proto.frameworks.crypten.onnx_model_pb2 import OnnxModel as OnnxModelPB
+
+
+class OnnxModel(AbstractSendable):
+    def __init__(
+        self,
+        serialized_model: bytes = None,
+        id: int = None,
+        owner: "syft.workers.AbstractWorker" = None,
+        tags: List[str] = None,
+        description: str = None,
+    ):
+        super(OnnxModel, self).__init__(id, owner, tags, description, None)
+        self.serialized_model = serialized_model
```

Comment:
Shouldn't we also store the model's weights as discussed in #3776 ?

Reply:
Based on our discussion, I will create another PR for doing that -- will link it to #3776 when doing that